dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -86,18 +88,18 @@ sub new {
 
 =pod 
 
-=head3 database($dbh, $meta, $update, $tarType, $tarLog, $DCMmd5, ...)
+=head3 database($dbh, $meta, $update, $tarType, $tarLog, $DCMmd5, $Archivemd5, $Archive, $neurodbCenterName)
 
-Inserts or updates the tarchive tables.
+Inserts or updates the C<tarchive> tables.
 
 INPUTS:
   - $dbh              : database handle
   - $meta             : name of the .meta file
-  - $update           : set to 1 to update tarchive entry, 0 otherwise
+  - $update           : set to 1 to update C<tarchive> entry, 0 otherwise
   - $tarType          : tar type version
   - $tarLog           : name of the .log file
   - $DCMmd5           : DICOM MD5SUM
-  - $Archivemd5       : tarchive MD5SUM
+  - $Archivemd5       : DICOM archive MD5SUM
```

Comment:
should this be `MD5 Sum` like you did in other scripts? I personally don't care, but we need to be consistent (and I assume one is the correct one, technically).

Reply:
good catch!